.github/workflows/continous-integration.yml

```diff
@@ -192,9 +192,16 @@ jobs:
     needs: [changes]
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8]
-        test: [test-non-training, test-training]
+        test:
```

Comment:
another note: I moved the `test` part up so that this shows up first when looking at the `Checks` as i think this more interesting than Python Version / OS